user-operator/src/test/java/io/strimzi/operator/user/model/acl/SimpleAclRuleResourceTest.java

```diff
@@ -283,4 +284,23 @@ public void testFromCrdToKafkaResourcePatternForTransactionalIdResource()  {
         ResourcePattern expectedKafkaTransactionalIdResourcePattern = new ResourcePattern(ResourceType.TRANSACTIONAL_ID, "my-transactionalId", PatternType.LITERAL);
         assertThat(SimpleAclRuleResource.fromCrd(resource).toKafkaResourcePattern(), is(expectedKafkaTransactionalIdResourcePattern));
     }
+
+    @Test
+    public void testFromCrdToKafkaResourceWithoutName()  {
```

Comment:
```suggestion
    public void testFromCrdToKafkaResourceWithoutNameThrows()  {
```
Minorly makes clearer the intent in my opinion.